README.md

```diff
@@ -54,7 +54,7 @@ Read the [Getting started page](https://getbootstrap.com/docs/4.3/getting-starte
 ## Status
 
 [![Slack](https://bootstrap-slack.herokuapp.com/badge.svg)](https://bootstrap-slack.herokuapp.com/)
-[![Build Status](https://img.shields.io/travis/twbs/bootstrap/master.svg)](https://travis-ci.org/twbs/bootstrap)
+[![Build Status](https://img.shields.io/circleci/project/gh/twbs/bootstrap/master.svg)](https://circleci.com/gh/twbs/bootstrap)
```

Comment:
@XhmikosR, I think this should be:
```suggestion
[![Build Status](https://img.shields.io/circleci/project/github/twbs/bootstrap/master.svg)](https://circleci.com/gh/twbs/bootstrap)
```

Reply:
Right, it's from the time I was using the wrong link :P